veniq/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Veniq uses Machine Learning to analyze source code, find possible refactorings, and suggest those that seem optimal
+"""
+
+__version__ = '0.0.0'
+__author__ = 'Yegor Bugayenko'
```

Comment:
why Yegor Bugayenko?

Reply:
cause he is the main person, the same thing was in aibolit

Reply:
Please change it )

Reply:
What should we write here?

Reply:
fixed